prediction_market_agent/agents/replicate_to_omen_agent/omen_replicate.py

```diff
@@ -66,7 +65,11 @@ def omen_replicate_from_tx(
         10 if market_type == MarketType.POLYMARKET else 1000,
         market_type,
         filter_by=FilterBy.OPEN,
-        sort_by=sort_by,
+        sort_by=(
+            SortBy.NONE
+            if market_type == MarketType.POLYMARKET
```

Comment:
Polymarket doesn't allow sorting by closing soonest

Reply:
I would suggest sorting by Volume - this way, we have the "most interesting" (or relevant) markets.

Maybe for another ticket - when creating the market, can we create it with different odds than the default 50/50? I would say we should copy the odds from the source market.

Reply:
It's further sorted by volume just below this: 

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/13ca49d1-efb1-4072-a496-8993c5af299d)

> Maybe for another ticket - when creating the market, can we create it with different odds than the default 50/50? I would say we should copy the odds from the source market.

Yes that's possible

Reply:
@coderabbit create a new issue out of this discussion

Reply:
Referenced question 2 [here](https://github.com/gnosis/prediction-market-agent-tooling/issues/406). Closing this.